src/current/v23.1/migrate-from-mysql.md

```diff
@@ -44,173 +68,349 @@ Example usage:
+              language VARCHAR(30) COLLATE en_US DEFAULT '' NOT NULL,
+              isofficial countrylanguage_isofficial_enum
+              DEFAULT 'F'
+              NOT NULL,
+              percentage DECIMAL(4,1) DEFAULT '0.0' NOT NULL,
+              PRIMARY KEY (countrycode, language),
+              INDEX countrycode (countrycode),
+              CONSTRAINT countrylanguage_ibfk_1
+                     FOREIGN KEY (countrycode) REFERENCES country (code)
+       )
+       ~~~
 
-~~~
-       job_id       |  status   | fraction_completed |  rows  | index_entries | system_records |  bytes
---------------------+-----------+--------------------+--------+---------------+----------------+----------
- 383839294913871873 | succeeded |                  1 | 300024 |             0 |              0 | 11534293
-(1 row)
-~~~
+       Click **Save**.
 
-### Import a table from a table dump
+       This is an optional workaround to reduce warnings during [data validation](#step-3-validate-the-migrated-data), due to a difference in how MySQL and CockroachDB handle case sensitivity in strings. For more details, see the [MOLT Verify](molt-verify.html#limitations) documentation.
 
-The following examples assume you [dumped one table](#dump-one-table-at-a-time).
+1. Click [**Migrate Schema**](../cockroachcloud/migrations-page.html?filters=mysql#migrate-the-schema) to create a new {{ site.data.products.serverless }} cluster with the converted schema. Name the database `world`.
 
-The simplest way to import a table dump is to run [`IMPORT`][import].  It reads the table data and any `CREATE TABLE` statements from the file:
+       You can view this database on the [**Databases** page](../cockroachcloud/databases-page.html) of the {{ site.data.products.db }} Console.
+       
+1. Open a SQL shell to the CockroachDB `world` cluster. To find the command, open the **Connect** dialog in the {{ site.data.products.db }} Console and select the `world` database and **CockroachDB Client** option. It will look like:
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-> CREATE DATABASE IF NOT EXISTS employees;
-> USE employees;
-> IMPORT TABLE employees FROM MYSQLDUMP 'https://s3-us-west-1.amazonaws.com/cockroachdb-movr/datasets/employees-db/mysqldump/employees.sql.gz';
-~~~
+       {% include_cached copy-clipboard.html %}
+       ~~~ shell
+       cockroach sql --url "postgresql://{username}@{hostname}:{port}/world?sslmode=verify-full" 
+       ~~~
 
-~~~
-       job_id       |  status   | fraction_completed |  rows  | index_entries | system_records |  bytes
---------------------+-----------+--------------------+--------+---------------+----------------+----------
- 383855569817436161 | succeeded |                  1 | 300024 |             0 |              0 | 11534293
-(1 row)
-~~~
+1. To optimize import performance, Cockroach Labs recommends removing indexes prior to [loading data](#step-2-load-the-mysql-data) and recreating them afterward.
```

Comment:
Link to https://www.cockroachlabs.com/docs/stable/import-performance-best-practices?